about.html

```diff
@@ -0,0 +1,160 @@
+              <a href="" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Join</a>
+          </li>
+             <li class="nav-item labelT">
+            <a href="" class="nav-link">WFH Remoters..</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+          
+      </div>
+      </nav>
+  </header>
+
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH Work from Home"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             
+             </div>
+             <div class="mid-label">
+              <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/R.png" alt="logo"/></a>
+              </div>
+             <div class="footer-content">
+    
+              <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3>
+              <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+            
+            
+            </div>
+          </div>
+
+             <div class="middle">
+              <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+              <h3>REMOTE DEV's GLOBAL SUMMIT ~2022</h3>
+              <p>
+                Kampala, Uganda (The Pearl of AFRICA) <br />
+                Its has never been better
+              </p>
+            
+            
+            </div>
+          </footer>
+             <div class="past">
+                <h2 class="past-header">See the past Global Summits</h2> <hr>
+                <p class="desc">Take a look at the last two global summits which took place in Buenos Aires and in warsaw</p>
+               <div class="past-img">
+                
+                <img src="images/conf.jpg" alt="" class="pas"><div class="center">
+                  <h4>2020</h4><p>Remote Global summit 2020 warsaw</p></div>
+                  <img src="images/devConf.png" alt="" class="pas">
+                  <div class="center1">
+                    <h4>2021</h4><p>Remote Global summit 2021 Buenos Aires</p></div>
+              </div>
+             </div>
```

Comment:
- [ ] The past conferences section has some issues with text orientation and font, the background image needs the `#EC5242` color with it's `opaqueness` set to less than 1. So that the background image is also apparent. 

Behance Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/183242443-37d2d5ca-960a-4e59-b477-29d4a0fd865a.png)  |  ![image](https://user-images.githubusercontent.com/60618877/183242465-86325184-35c5-44db-8dea-3085bb876651.png)